wix-custom-labels/src/index.js

```diff
@@ -0,0 +1,34 @@
+const isEqual = require('lodash.isequal')
+const CUSTOM_LABELS_KEY = 'customLabels'
+const CUSTOM_LABELS_TAG = 'customlabels'
+
+exports.defineTags = function(dictionary) {
+  dictionary.defineTag(CUSTOM_LABELS_TAG, {
+      mustHaveValue: true,
+      onTagged: function (doclet, tag) {
+        doclet.customLabels = []
+        tag.value.split(' ').forEach(customLabel => {
+          doclet.customLabels.push({'id': customLabel})
```

Comment:
Why do you crate an array of objects `[{id: 'foo'}, {id: 'bar'}]` instead of array of strings `['foo', 'bar']`?

Reply:
because this is what the viewer is expecting for.

Reply:
Can we change it on the viewer's side? I understand you already made some changes there

Reply:
the change in the viewer is to expect it at extra key.
someService.extra.customLabels instead of someService.customLabels.